.github/workflows/generate-code.yml

```diff
@@ -0,0 +1,87 @@
+name: Generate OpenAPI based code
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: Generate OpenAPI based code
+    runs-on: ubuntu-latest
+
+    steps:
+      # Setup
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Update submodules
+        run: git submodule update --remote --recursive
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
```

Comment:
probably we can remove this. python is installed in default github action runner. (sorry I can't find url...)

Reply:
https://github.com/actions/runner-images/blob/c21e3961c52a5611ee44c3d33b1b11ac7a4636b7/images/ubuntu/Ubuntu2404-Readme.md?plain=1#L27